projects/zilswap/project.md

```diff
@@ -1,10 +1,22 @@
 ---
 name: ZilSwap
-description: The premier DEX on the Zilliqa blockchain.
+description: The DeFi hub of Zilliqa.
+mobile_friendly: y
+smart_contract_addresses: zil1gkwt95a67lnpe774lcmz72y6ay4jh2asmmjw6u,zil1p5suryq6q647usxczale29cu3336hhp376c627
 categories: exchange,nft,web3,dex,api,staking,defi,dapp,app,sdk,oracle,token,governance,dao,infra
-status: Live
+status: live
 twitter: https://twitter.com/ZilSwap
 website: https://zilswap.io/
-discord: https://discord.gg/SMzzQ2sUMN
-telegram: False
+discord: https://discord.gg/ZilSwap
 ---
+full_description: 
```

Comment:
@xyn1nja over multiline values try wrapping it in quotes, This current file is breaking the structure.

Reply:
updated, please review, thanks.